lumicks/pylake/kymo.py

```diff
@@ -155,12 +165,13 @@ def _plot(self, image, **kwargs):
             # pixel center aligned with mean time per line
             extent=[-0.5 * linetime, duration - 0.5 * linetime, size_calibrated, 0],
             aspect=(image.shape[0] / image.shape[1]) * (duration / size_calibrated),
+            cmap=linear_colormaps[channel],
```

Comment:
Is there a reason you prefer pulling this into both functions, instead of having it at the `_plot_color` level like we did before?

I have a mild preference for having it in the other location, since changing it once will change it in both places in that case.